authn-authz/overview.md

```diff
@@ -2,10 +2,8 @@
 
 Tanzu Application Platform 1.1 includes:
 
-- Documentation for [integrating with your existing identity management solution](integrating-identity.md).
```

Comment:
I don't think the feedback was asking us to remove _Integrating with your existing identity management solution_. So you should bring this line back.

I think the feedback was saying to remove _Documentation for Logging in using Pinniped_ from the overview to give it less emphasis. But then paste the link inside _integrating with your existing identity management solution_ and also create the same parent-child relationship in the TOC.

Reply:
re-added the link